### ISSUE TITLE
Rule option proposal: indent { ParensAfterFunctionIdentifierAndNewline }

### ISSUE THREAD

**Issue by skylize** [eslint/eslint#11118]
**What rule do you want to change?**
indent

**Does this change cause the rule to produce more or fewer warnings?**
depends on usage (fewer warnings in *my* code 😄 )

**How will the change be implemented? (New option, new default behavior, etc.)?**
new option

**Please provide some example code that this change will affect:**

```js
const result = myFunction
  ( param1 )
  ( param2 )
```

**What does the rule currently do for this code?**

Expected indentation of 0 spaces but found 2.

**What will the rule do after it's changed?**

Report no error.

**Are you willing to submit a pull request to implement this change?**

I wish I had the slightest clue where to start. 😦
